### ISSUE TITLE
Pluralization with EF Core and EF Core Power Tools

### ISSUE THREAD

**Issue by ErikEJ** [ErikEJ/erikej.github.io#6]

**Comment by AdalbertKhoolsaat** [comment]
Coming here with a question after reading [the related blog post](https://erikej.github.io/efcore/2020/04/26/ef-core-pluralization.html). 

My situation is that I am dealing with tables, views and columns with non-English names (Dutch to be precise). A Humanizer.Core.nl package is available, but as of yet I am unable to get the custom pluralization example to work using EF Core Scaffold-DbContext. Besides, I much prefer using EF Core Power Tools for the Reverse Engineering task for various reasons. 

Is there currently any way to override the internally used Humanizer.Core package to facilitate for correct non-English pluralization when using EF Core Power Tools?

**Comment by ErikEJ** [comment]
@AdalbertKhoolsaat No, currently not. Maybe propose a feature in the GitHub repo for the tool?
